### DISCUSSION TITLE
Fractional Scaling

### DISCUSSION THREAD

**Discussion by 0x9394** [elementary/triage#366] [Ideas]
`fractional scaling` is available in upstream OS for a long time.
I don't run into problems when try ubuntu with 1.75 scaling ( tried mint either ).

text enlarge is looks really bad when I use notebook's 2K screen. (1x scale is too small for items, 2x scale is too dam large)

thanks for reading.

**Comment by doctor-amaton** [comment]
I had the same concern before moving into 7.1, but I noticed that if you change the text to 1.25 it does enlarge everything else, in my 1920x1200 13" monitor it produces the same effect as 125% scaling on Windows, even Chrome respects the object size, have you tried that out?

**Reply by 0x9394** [reply]
yes, I have, tried all text enlarge options with 1x scale. all looks bad .
notebook is 13.3 inch, and external screen is 27 inch with 4K. tried all option. now is using 2X scale with external screen.

**Comment by danirabbit** [comment]
Closing as completed since we now support fractional scaling in the secure session as of OS 8.1!
